mote/static/css/mote.css

```diff
@@ -54,7 +54,7 @@
   width: 80%;
 }
 
-.log-display pre { white-space: pre-wrap; }
+.log-display pre { white-space: pre-wrap; word-break: keep-all; }
```

Comment:
The last preview at https://github.com/fedora-infra/mote/pull/147#issuecomment-721194503 - had both `pre-wrap` and `keep-all` as `white-space` and `word-break` configs. Looks good to me. :smile: